spec/lib/bolt_spec/project.rb

```diff
@@ -3,6 +3,7 @@
 module BoltSpec
   module Project
     def with_project(name = 'project')
+      @project_name = name
```

Comment:
```suggestion
```

I don't think this is necessary if we ensure the name is written to the config file in `with_project`.